### PR TITLE
change docs and adjust dashboard for v2 alpha

### PR DESCRIPTION
**Pull request by SantoDE**
### What does this PR do?

Adds some notice about the WIP Dashboard

### Motivation
Make it a bit more user friendly?

### More

- [ ] Added/updated tests
- [x] Added/updated documentation

### Additional Notes

Co-authored-by: Jean-Baptiste Doumenjou <jb.doumenjou@gmail.com>

### CODE REVIEW

File: docs/content/operations/dashboard.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ See What's Going On
 
 The dashboard is the central place that shows you the current active routes handled by Traefik. 
 
+<aside class="notice">
+   Currently, the dashboard is in a Work In Progress State while beeing reconstructed for v2. Therefore, the dashboard is currently not working
```
Comment (mpl) [inline docs/content/operations/dashboard.md]: missing period at the end of the sentence.

Comment (SantoDE) [reply docs/content/operations/dashboard.md]: Fixed :)

File: docs/content/operations/dashboard.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ See What's Going On
 
 The dashboard is the central place that shows you the current active routes handled by Traefik. 
 
+??? note "Dashboard WIP"
+   Currently, the dashboard is in a Work In Progress State while beeing reconstructed for v2.
```
Comment (jbdoumenjou) [inline docs/content/operations/dashboard.md]:
```suggestion
   Currently, the dashboard is in a Work In Progress State while being reconstructed for v2. 
```

Comment (jbdoumenjou) [inline docs/content/operations/dashboard.md]:
```suggestion
   Currently, the dashboard is in a Work In Progress State while being reconstructed for v2. 
```

**Pull request merged**